### PR TITLE
Adds three unique kinetic accelerator modules to necropolis chests

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Three unique Kinetic Accelerator modules will now appear in necropolis chests.
/:cl:

Rapid Repeater, which increases the Kinetic Accelerator's cooldown by 14 deciseconds, but divides the KA's cooldown by 4 if it hits a living target or mineral turf. Without cooldown modules, this means it has a cooldown of 3 seconds if you miss, but a cooldown of 7.5 seconds otherwise.
Lifesteal Crystal, which heals the firer for 2.5 health when they hit a living target. Also comes with a cult bedsheet.
Resonator Blast, which leaves and detonates resonator blasts like a resonator. Blasts detonated do 25%(15) of normal blast damage if detonated by this. It won't create resonator blasts on mineral turfs, though.

The repeater replaces the SORD... because oh god it's the literal worst reward.
The lifesteal crystal replaces the cult robes and bedsheet.

### CODE REVIEW

**Comment by Repukan** [comment]
Lifesteal crystal should have a 50% chance of activating bloodlust which gives a 20% chance of healing 40% of total damage done for 2d2 seconds

**Comment by ChangelingRain** [comment]
We don't assign damage well enough for that to be easy or practical to code.

**Comment by Repukan** [comment]
It's like you hate fun

**Comment by Iamgoofball** [comment]
@ChangelingRain I know a way, hang on

**Comment by ShadowDimentio** [comment]
The repeater is hysterically expensive for being a sidegrade instead of an upgrade. Reduce it to 30 or 20.

**Comment by ChangelingRain** [comment]
If you don't miss with the repeater, it's better than having 3 cooldown modules and still gives you space for two range modules. At best, I'll drop it to 45, so you can do range+one 30 cost module.

**Comment by jughu** [comment]
Keep the sord in but make it 0.000000001% like arcade pulse rifles RARE REWARDS

**Comment by YakumoChen** [comment]
but the SORD is funny

**Comment by YakumoChen** [comment]
IMO not every chest needs to have usable loot. Having trash loot adds rarity to the good loot.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -68,6 +66,9 @@
 			new /obj/item/weapon/spellbook/oneuse/summonitem(src)
 		if(26)
 			new /obj/item/book_of_babel(src)
+		if(27)
```
Comment (lzimann) [inline code/modules/mining/lavaland/necropolis_chests.dm]: We really should start using defines for these.

**Comment by adamkad1** [comment]
Repeater seems confusing, does it reward hitting or not? also can you fix explosive mining and hybrid mods? explosive mining doesnt work

**Comment by Repukan** [comment]
Explosive from the vendor makes a non mining aoe that hits enemies. You can see this by shooting legions.

The RnD one only mines afaik, and the hybrid one from the chests does both, mine and damage.

**Comment by ChangelingRain** [comment]
Repeater basically means you have slightly better than 3 cooldown mods if you don't miss.

**Comment by adamkad1** [comment]
Repukan, the proper with mining explosion is, it does not work. it does not mine 3x3, it does NOTHING

**Pull request merged**